src/index.scss

```diff
@@ -1,3 +1,16 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter&family=Philosopher&display=swap');
+
+// Font Variables
+$inter: 'Inter', sans-serif;
+$philosopher: 'Philosopher', sans-serif;
+
+//Colour Variables
```

Comment:
Typo: Color

Reply:
I use the British English version of spelling words (e.g colour instead of color or humour instead of humor)

Reply:
I see now, I fixed it. Thank you :)